packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.ts

```diff
@@ -70,6 +70,11 @@ export abstract class XYChartComponent extends TimeseriesChartComponent
         return this.configuration.legendPlacement === LegendPlacement.Right;
     }
 
+    /** Checks if the series is interactive or if the dataSource has a link property */
+    public isSeriesInteractive(legendSeries: IChartAssistSeries<IAccessors>): boolean {
+        return !(legendSeries?.link === undefined) || !(legendSeries?.secondaryLink === undefined) || this.seriesInteractive;
+    }
+
     public onLegendClick(legendSeries: IChartAssistSeries<IAccessors>, event: MouseEvent) {
```

Comment:
It seems like this method should be removed now that we're handling the interaction emission with onPrimaryDescClick